### ISSUE TITLE
build error in caffe-fact-rcnn

### ISSUE THREAD

**Issue by jwnsu** [Eniac-Xie/faster-rcnn-resnet#1]
Env: Ubuntu 16.04 with latest CUDA 8.0 cuDNN 5.1 

Copied Makefile.config from working py-faster-rcnn directory. make reports following error and stops:
------
CXX src/caffe/data_reader.cpp
In file included from ./include/caffe/util/device_alternate.hpp:40:0,
                 from ./include/caffe/common.hpp:19,
                 from src/caffe/data_reader.cpp:6:
./include/caffe/util/cudnn.hpp: In function ‘void caffe::cudnn::createPoolingDesc(cudnnPoolingStruct**, caffe::PoolingParameter_PoolMethod, cudnnPoolingMode_t*, int, int, int, int, int, int)’:
./include/caffe/util/cudnn.hpp:127:41: error: too few arguments to function ‘cudnnStatus_t cudnnSetPooling2dDescriptor(cudnnPoolingDescriptor_t, cudnnPoolingMode_t, cudnnNanPropagation_t, int, int, int, int, int, int)’
         pad_h, pad_w, stride_h, stride_w));
                                         ^
./include/caffe/util/cudnn.hpp:15:28: note: in definition of macro ‘CUDNN_CHECK’
     cudnnStatus_t status = condition; \
                            ^
In file included from ./include/caffe/util/cudnn.hpp:5:0,
                 from ./include/caffe/util/device_alternate.hpp:40,
                 from ./include/caffe/common.hpp:19,
                 from src/caffe/data_reader.cpp:6:
/usr/local/cuda/include/cudnn.h:803:27: note: declared here
 cudnnStatus_t CUDNNWINAPI cudnnSetPooling2dDescriptor(
                           ^
Makefile:563: recipe for target '.build_release/src/caffe/data_reader.o' failed
make: *** [.build_release/src/caffe/data_reader.o] Error 1

**Comment by Eniac-Xie** [comment]
Hi, can you successfully build the caffe-fast-rcnn of [rbgirshick's py-fast-rcnn](https://github.com/rbgirshick/py-faster-rcnn)? I only mdified the eltwise layer and batchnorm layer in caffe-fast-rcnn.

**Comment by jwnsu** [comment]
Saw you only made 3 file changes. So I copied over a working caffe-fast-rcnn, then delta the changes. It's resolved now.

Model file from pan.baidu.com takes forever to download in US though, poor baidu service.

**Comment by Eniac-Xie** [comment]
@jwnsu Add OneDrive download link in README.

**Comment by jwnsu** [comment]
Thanks! Got models in just seconds from onedrive.

**Comment by askerlee** [comment]
Following @jwnsu 's way, I copied eltwise_layer.* and batch_norm_layer.* from caffe-fast-rcnn/src/caffe/layers/ to a working caffe-fast-rcnn. Now caffe compiles smoothly.
